[MTM-51095]: Announcement about status code change for MO delete API (#179)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/announcements-10-17-0.md

As a consequence, if a request is supposed to return child assets, devices and additions, a parameter `?withChildren=true` has to be provided in the query.
This change is necessary to improve the Inventory API performance.

> ##### Breaking change in the Inventory API -- change of the return code for delete of managed object

Reviewer: ```suggestion
##### Breaking change in the Inventory API -- change of the return code for the deletion of managed objects
```

> As of release 10.19+, when delete of managed object and its dependencies doesn't finish right away, the platform will return 202 (Accepted) http status code instead of 204 (No content).

Reviewer: ```suggestion
As of release 10.19+, if the deletion of a managed object and its dependencies doesn´t finish immediately, the platform will return a 202 (Accepted) HTTP status code instead of 204 (No content).
```

> A return code will be still 204 if the control can be returned immediately.

Reviewer: ```suggestion
The return code will still be 204 if the control can be returned immediately.
```

> The change is required to make API consistent with HTTP protocol semantics (asynchronous request).

Reviewer: ```suggestion
This change is required to make the API consistent with the HTTP protocol semantics (asynchronous request).
```
